content/platforms/kubernetes/reference/redis_enterprise_database_api.md

```diff
@@ -0,0 +1,268 @@
+---
+Title: Redis Enterprise Database API
```

Comment:
```suggestion
Title: Redis Enterprise Database - Custom Resource Specification
```

Reply:
This document is a more complete version of db-options.md. We should not have both.